pkgs/tools/networking/unbound/default.nix

```diff
@@ -100,12 +107,21 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
+  postPatch = ''
+    substituteInPlace Makefile.in \
+      --replace "\$(DESTDIR)\$(PYTHON_SITE_PKG)" "$out/${python.sitePackages}"
```

Comment:
This should also probably be made `lib.optionalString` else it wouldn't be possible to build an unbound without supplying a `python` to fill in this value. There could also a small risk of accidentally leaving a reference to python in the output even in non-python builds.

Reply:
Nice catch